cli/setup_test.go

```diff
@@ -282,12 +284,22 @@ func TestInstallDemoCertificateLocalTrust(t *testing.T) {
 	assert.NoError(t, err)
 
 	// Verify that the demo cert was installed in the local trust
-	_, err = os.Stat(getLocalTrustMenderCertPath())
+	_, err = os.Stat(DefaultLocalTrustMenderDir)
 	assert.NoError(t, err)
 	crtSource, err := ioutil.ReadFile(getMenderDemoCertPath())
 	assert.NoError(t, err)
-	crtInstall, err := ioutil.ReadFile(getLocalTrustMenderCertPath())
-	assert.NoError(t, err)
-	same := bytes.Equal(crtSource, crtInstall)
-	assert.True(t, same)
+
+	crtInstall, err := ioutil.ReadDir(DefaultLocalTrustMenderDir)
+	assert.Equal(t, 2, len(crtInstall))
+	for _, entry := range crtInstall {
+		checkCrtInstall(t, path.Join(DefaultLocalTrustMenderDir, entry.Name()), crtSource)
+	}
+}
+
+func checkCrtInstall(t *testing.T, cert string, demoCertContent []byte) {
+	assert.True(t, strings.HasPrefix(path.Base(cert), DefaultLocalTrustMenderPrefix))
+	content, err := ioutil.ReadFile(cert)
+	require.NoError(t, err)
+	assert.Greater(t, len(content), 10)
+	assert.True(t, bytes.Contains(demoCertContent, content))
```

Comment:
We could also verify that the first line has "BEGIN CERTIFICATE" and the last one "END CERTIFICATE"

Reply:
Good idea!